pkg/virt-launcher/virtwrap/stats/types.go

```diff
@@ -60,6 +60,8 @@ type DomainStats struct {
 	Cpu *DomainStatsCPU
 	// new, see below
 	Memory *DomainStatsMemory
+	// new, see below
```

Comment:
I know this comments exists for other fields, but I don't feel it's very useful.
I think that either state where is came from (e.g. `// omitted from libvirt-go: DomainJobInfo`) or remove it alltogether

Reply:
added the comment in the format you suggested, makes sense, tnx.